### PR TITLE
Feature/store management app

### PR DESCRIPTION
**Pull request by lilvolcanointhenighttime**
rebase to create feature/users_app

### CODE REVIEW

**Review by lilvolcanointhenighttime — COMMENTED** [review]
...

**Review by lilvolcanointhenighttime — COMMENTED** [review]
:)

**Comment by lilvolcanointhenighttime** [comment]
comment

File: .env.example [hunk 1]
```diff
@@ -5,12 +5,12 @@ DJANGO_ALLOWED_HOSTS=
 DJANGO_CORS_ALLOWED_ORIGINS=
 
 # ===== DATABASE =====
-DB_NAME_STORE_MANAGEMENT=exapp
+DB_NAME_STORE_MANAGEMENT=
 DB_NAME_PAYMENTS=
 DB_NAME_USERS=
 DB_USER=
 DB_PASSWORD=
-DB_HOST=db
```
Comment (lilvolcanointhenighttime) [inline .env.example:13]: 12

File: README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+![first_db_structure](https://github.com/lilvolcanointhenighttime/online_shop_backend/assets/125156127/ae4e7689-411a-4d4a-bfc8-bd7a60cea2ac)
```
Comment (lilvolcanointhenighttime) [inline README.md:1]: cool

**Review by reviewerlilvolcanointhenighttime — APPROVED** [review]
approved

**Review by reviewerlilvolcanointhenighttime — APPROVED** [review]

**Review by reviewerlilvolcanointhenighttime — COMMENTED** [review]
cool

**Review by reviewerlilvolcanointhenighttime — CHANGES_REQUESTED** [review]
asd

File: .env.example (hunk 1 above)
Comment (reviewerlilvolcanointhenighttime) [reply .env.example:13]: ads

**Review by reviewerlilvolcanointhenighttime — APPROVED** [review]
<3

**Pull request merged**